quickwit-backward-compat/test-data/index-metadata/unversioned.json

```diff
@@ -2,7 +2,7 @@
   "checkpoint": {
     "0000000000": "0000000042"
   },
-  "index_config": {
+  "doc_mapper": {
```

Comment:
I didn't see any `serde(alias = ...)` anywhere, so are we breaking backward compatibility?